manifests/server/startup.pp

```diff
@@ -0,0 +1,90 @@
+# == Define: dns::server::startup
```

Comment:
How about class dns::server::defaults? The definition of **defaults_file** will be used in the code to set the filename.